### ISSUE TITLE
Master/Detail Lookup Textual Fields not showing

### ISSUE THREAD

**Issue by rgilsimoes** [volkanceylan/Serenity#1231]
Hi, 
I have this GridEditor in a Dialog:
![image](https://cloud.githubusercontent.com/assets/20357414/18781016/f7e5c33c-8177-11e6-8a67-287a4b235215.png)

Columns 3 and 9 are set as Expressions on Row.cs and are TextualFields from column 2 and 8.

I've set columns 2 and 8 as LookupEditor in the Form.cs and Row.cs but when I add a new record the GridEditor only displays the IdField (2 and 8) and not the TextualField (3 and 9).

I've also included those columns names on MasterDetailRelation "IncludeColumns"

**Comment by volkanceylan** [comment]
If you don't have this fields in your detail row, and don't specify exact names, it won't work. It's really tiring to resolve same issue again and again.

**Comment by volkanceylan** [comment]
When you add a new record, you fill in those fields yourself, in validateEntity. It is specified in guide, and other 20+ issues before, come on.

**Comment by rgilsimoes** [comment]
It doen's make any sense to me anyway. If we are supose to be doing "declarative programming"...

**Comment by volkanceylan** [comment]
Well, find a way to make in memory grid to work with SQL, and make it work declaratively, and share your pull request.

**Comment by rgilsimoes** [comment]
Just another thing I may be missing on this one:
I have added the required code on "validateEntity" and it is ok except for an EnumType Field.

I have a "Status" field that is a custom EnumType and in the Grid it displays the Enum Text but after adding a new record to the grid it dysplays the Enum Value instead.

Tried to update row.Status to the respective Text but it says incompatible types.

Thanks,
